pkg/systemd/activation.go

```diff
@@ -0,0 +1,40 @@
+package systemd
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Determine if podman is running under the socket activation protocol
+func SocketActivated() (int, bool) {
```

Comment:
The `int` doesn't seem to be used. Can we remove it?

Reply:
I did.  If we need it in the future, we can always add it back.